v23.1/restoring-backups-across-versions.md

```diff
@@ -16,17 +16,16 @@ Since CockroachDB considers the cluster version when running a backup, this page
 
 ## Support for restoring backups into a higher version
 
-Cockroach Labs supports restoring a backup taken on a cluster on a specific major version into a cluster that is on the same version or the next major version. Therefore, when you're upgrading your cluster from version N to N+1, if you took a backup on major version N, you can restore it to your cluster on either major version N or N+1.
+Cockroach Labs supports restoring a backup taken on a cluster on a specific major version into a cluster that is on the same version or the next major version. Therefore, when upgrading your cluster from version N to N+1, if you took a backup on major version N, you can restore it to your cluster on either major version N or N+1. Restoring backups outside major version N or N+1 is not supported. 
 
-For example, backups taken on v21.1.x can be restored into v21.2.x. Backups taken on v21.2.x can be restored into v22.1.x. The following table outlines this support:
+For example, backups taken on v22.1.x can be restored into v22.2.x. Backups taken on v22.2.x can be restored into v23.1.x. The following table outlines this support:
 
 Backup taken on version   | Restorable into version
 --------------------------+--------------------------
-20.1.x                    | 20.2.x
```

Comment:
In the Restorable into version column, we should also include the same version in addition to the next version.

Reply:
Added both versions to be restored so it will be for example: 20.1.x, 20.2.x